dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/security/oidc/DhisOidcUser.java

```diff
@@ -174,7 +174,7 @@ public boolean hasAnyAuthority(Collection<String> auths) {
 
   @Override
   public boolean isAuthorized(String auth) {
```

Comment:
does `hasAnyAuthority` need to be updated also?

Reply:
I think so